### ISSUE TITLE
2.3

### ISSUE THREAD

**Issue by sonali-sr** [rebeccajohnson88/PPOL564_slides_activities#58]
### 2.3 Estimate a topic model using those preprocessed words (5 points)

**A.** Going back to the preprocessed words from part 2.1, estimate a topic model with 3 topics, since you want to see if the unsupervised topic models recover different themes for each of the three manually-labeled topics (civil rights; hate crimes; project safe childhood). You have free rein over the other topic model parameters beyond the number of topics.

**B.** After estimating the topic model, print the top 15 words in each topic.

<img width="876" alt="2 3B" src="https://user-images.githubusercontent.com/90288860/197817366-15fa9a28-9f74-419e-ac2f-0e462365702c.PNG">

**Hints and Resources:**
- Same topic modeling resources linked to earlier
- Make sure to use the `random_state` argument within the model so that the numbering of topics does not move around between runs of your code

**Comment by qinip** [comment]
Hi Professor, my three top word lists have many overlapping ones with your result here, but in different orders and probability scores. Also, results vary as I change the `random_state` value. We're not going to have the exact output as your screenshot here, right? Thanks!

**Comment by rebeccajohnson88** [comment]
Yep totally fine for output to differ since it depends on preprocessing and the random state! Thanks for checking!
